gwpy/segments/flag.py

```diff
@@ -494,7 +494,7 @@ def query_dqsegdb(cls, flag, *args, **kwargs):
         for start, end in qsegs:
             # handle infinities
             if float(end) == +inf:
-                end = to_gps('now').seconds
+                end = to_gps('now').gpsSeconds
```

Comment:
Can you implement this change to make the operation object agnostic?

```suggestion
                end = int(to_gps('now'))
```